src/class-block-converter.php

```diff
@@ -315,21 +320,44 @@ public function remove_image_args( $url ): string {
 		// Split url.
 		$url_parts = wp_parse_url( $url );
 
-		return $url_parts['scheme'] . '://' . $url_parts['host'] . $url_parts['path'];
+		// Negotiate scheme.
+		$scheme = $url_parts['scheme'] ?? '';
+		if ( empty( $scheme ) ) {
+			$home = get_option( 'home' );
```

Comment:
I'm wondering whether this should use the URL to the uploads directory via `wp_get_upload_dir()`, not to the site frontend.